frontend/src/v4/routes/viewerGui/components/bim/bim.component.tsx

```diff
@@ -178,13 +179,15 @@ export class Bim extends PureComponent<IProps, any> {
 					<Tab label={isV5() ? 'All data' : 'All'} />
 					<Tab label="Starred" />
 				</Tabs>
-				<ViewerPanelContent>
-					<Container>
-						{this.renderMetadata()}
-						{this.renderEmptyState(!areFiltersActive && !hasMetadata)}
-						{this.renderNotFound(areFiltersActive && !hasMetadata)}
-					</Container>
-				</ViewerPanelContent>
+				<ViewerScrollArea autoHeight autoHeightMax={'100%'}>
+					<ViewerPanelContentComponent>
+						<Container>
```

Comment:
would it work to change ViewerPanelContent to be the scroller ?

Reply:
Yes, it works. I changed the code to match this